packages/cli/commands/add/index.ts

```diff
@@ -410,20 +367,20 @@ export async function runAddCommand(
 		}
 	}
 
+	// add verifications and inter-addon deps
+
 	// add inter-addon dependencies
 	for (const { addon } of selectedAddons) {
-		workspace = await createWorkspace(workspace);
-
-		const setups = selectedAddons.map(({ addon }) => addon);
-		const setupResult = setupAddons(setups, workspace)[addon.id];
+		workspace = virtualWorkspace || (await createWorkspace({ ...workspace }));
```

Comment:
not sure this is the right call here. Previously we `createWorkspace` after every run, so that if tailwind and vitest run (after each other), the vitest run should already know via the workspace that `tailwind` is already installed.

Or am I missing something here?

Reply:
Actually not sure why that works, but that seems to work